music21/musicxml/xmlToM21.py

```diff
@@ -2141,19 +2135,19 @@ def __init__(self, mxMeasure=None, parent=None):
         self.staffLayoutObjects = []
         self.stream = stream.Measure()
 
-        self.mxNoteList = [] # for accumulating notes in chords
-        self.mxLyricList = [] # for accumulating lyrics assigned to chords
-        self.nLast = None # for adding notes to spanners.
-        self.chordVoice = None # Sibelius 7.1 only puts a <voice> tag on the
-                        # first note of a chord, so we need to make sure
-                        # that we keep track of the last voice...
+        self.mxNoteList = []  # for accumulating notes in chords
+        self.mxLyricList = []  # for accumulating lyrics assigned to chords
+        self.nLast = None  # for adding notes to spanners.
+        self.chordVoice = None  # Sibelius 7.1 only puts a <voice> tag on the
```

Comment:
If you're unindenting the continuation of a comment, the whole comment should go before the definition so it's clear that it continues.